source/includes/connect/mongoclient.c

```diff
@@ -0,0 +1,23 @@
+#include <mongoc/mongoc.h>
+#include <bson/bson.h>
+
+int main (void) {
+
+  // start-connect-to-atlas-w-uri
+  mongoc_client_t *client;
+  mongoc_database_t *database;
+  char *uri;
+
+  mongoc_init ();
+
+  // Create a new client and connect to the server
+  uri = "mongodb+srv://<db_username>:<db_password>@<hostname/port>/?<options>";
```

Comment:
The standard connection string descrption only lists `mongodb://`, but the immediately following Atlas connection examples use `mongodb+srv://`. Recommend updating the connection string description to include SRV connection strings, which is what is used in example code.